rllib/evaluation/rollout_worker.py

```diff
@@ -339,14 +345,23 @@ def __init__(
                 DefaultCallbacks for training/policy/rollout-worker callbacks.
             input_creator: Function that returns an InputReader object for
                 loading previous generated experiences.
-            off_policy_estimation_methods: How to evaluate the policy performance.
-                Setting this only makes sense when the input is reading offline data.
-                Available options:
-                - "simulation" (str): Run the environment in the background, but use
+            off_policy_estimation_methods: Specify how to evaluate the current policy,
+                along with any optional config parameters.
+                This only has an effect when reading offline experiences
+                ("input" is not "sampler").
+                Available keys:
+                - {"simulation": None}: Run the environment in the background, but use
                 this data for evaluation only and not for learning.
-                - Any subclass (type) of the OffPolicyEstimator API class, e.g.
-                `ray.rllib.offline.estimators.importance_sampling::ImportanceSampling`
-                or your own custom subclass.
+                - {ope_method_name: {"type": ope_type, ...}} where `ope_type` can be:
```

Comment:
??

Reply:
Hm, I'm not sure how we can put this into the TrainerConfig objects though, since it's inside the optional config dict for the OPE, plus we want to support arbitrary Q models and corresponding arguments. We can chat about this tomorrow maybe?